### PR TITLE
fix: show the error message for text input widget

### PR DESCRIPTION
**Pull request by techbhavin**
## Description

> No error is thrown if one enters some text in the Input widget and then sets a Max Chars property such that it is less than the number of characters entered.

Fixes [#14886](https://github.com/appsmithorg/appsmith/issues/14886)

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- D&D a Input widget.
- Enter 20 characters.
- Not set a limit on the Max Chars property.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/ce/constants/messages.ts [hunk 1]
```diff
@@ -22,6 +22,8 @@ export const API_PATH_START_WITH_SLASH_ERROR = () => `Path cannot start with /`;
 export const FIELD_REQUIRED_ERROR = () => `This field is required`;
 export const INPUT_DEFAULT_TEXT_MAX_CHAR_ERROR = () =>
   `Default Text length must be less than Max Chars allowed`;
+export const INPUT_TEXT_MAX_CHAR_ERROR = () =>
+  `Input Text length must be less than Max Chars allowed`;
```
Comment (mohanarpit) [inline app/client/src/ce/constants/messages.ts:26]:
```suggestion
  `Input text length must be less than max chars allowed`;
```

Nit: Correcting the sentence casing here.

Also, is it possible to modify this error message to - Eg: "Input text length must be less than 140 characters"? In this example, 140 is the max chars property. This way, a user is able to accurately determine & fix the error in their input.

File: app/client/src/ce/constants/messages.ts [hunk 2]
```diff
@@ -22,6 +22,8 @@ export const API_PATH_START_WITH_SLASH_ERROR = () => `Path cannot start with /`;
...
 export const INPUT_DEFAULT_TEXT_MAX_NUM_ERROR = () =>
   `Default Text value must be less than Max number allowed`;
```
Comment (mohanarpit) [inline app/client/src/ce/constants/messages.ts:28]: I know this is not part of the scope, but can we apply the same logic to this error message as well? Correct the sentence casing and also add the max number allowed in the input.

File: app/client/src/ce/constants/messages.ts (hunk 1 above)
Comment (somangshu) [reply app/client/src/ce/constants/messages.ts:26]: Getting picked up in #14830

**Review by mohanarpit — APPROVED** [review]

**Comment by Tooluloope** [comment]
@techbhavin Failed DP

**Pull request merged**